core/src/main/scala/spinal/core/ScopeProperty.scala

```diff
@@ -5,65 +5,68 @@ import spinal.idslplugin.PostInitCallback
 import scala.collection.mutable
 import scala.collection.mutable.{ArrayBuffer, Stack}
 
-class ScopePropertyContext{
-  val mutableMap = mutable.HashMap[ScopeProperty[Any], Any]() //For stuff changing often
-  var immutableMap = scala.collection.immutable.HashMap[ScopeProperty[Any], Any]() //For things mostly static
+class ScopePropertyContext {
+  val mutableMap =
+    mutable.HashMap[ScopeProperty[Any], Any]() // For stuff changing often
```

Comment:
Maybe we can disable the 80 char par line ? Or relax it ?

Reply:
Yes: specify maxColumn in scalafmt.conf.

Reply:
How many chars would you prefer? Maybe an large enough number like 999.

Reply:
hmmm, maybe 160 ?

Reply:
Done